tsl/src/compression/create.c

```diff
@@ -585,7 +585,7 @@ create_compression_table(Oid owner, CompressColInfo *compress_cols)
  * Caller is expected to do this explicitly.
  */
 Chunk *
-create_compress_chunk_table(Hypertable *compress_ht, Chunk *src_chunk)
+create_compress_chunk_table(Hypertable *compress_ht, Chunk *src_chunk, Oid table_id)
```

Comment:
Not sure it's feasible, but would be more straightforward if this was two functions, create the table + attach it as a compressed chunks. When moving the chunk, you replace the "create the table" part with "fetch it from other node".

Reply:
I mean, now the function "something-something-create-table" has an argument that means "but don't actually create the table", and this might be confusing.

Reply:
I mean, can we move the `ts_chunk_create_table` out of  `create_compress_chunk_table`, and always pass `table_oid`? And call `create_compress_chunk_table` something like `create_compressed_chunk_metadata`. If not, makes sense to at least rename it somehow, because now it reads like `void create_table(bool do_not_create_table)` .

Reply:
This function is used in other places, like `compress_chunk`. I am not sure it is a good idea to touch them here and do bigger refactoring.

I think the problem is the function name, how about name it just `create_compress_chunk` instead?

Reply:
I've renamed it, hope it looks better now

Reply:
Yes, this name is better